.github/workflows/deploy-to-eks.yaml

```diff
@@ -16,45 +16,39 @@ on:
     - cron: "23 0 * * 2"
 jobs:
   preprocess-input:
-    if: ${{ github.event_name == 'workflow_dispatch' }}
     runs-on: ubuntu-22.04
     outputs:
       processed_bundle_versions: ${{ steps.process_bundle_versions.outputs.bundle_versions }}
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Why not using v4?

Reply:
@rgildein Was unsure of whether to update this at all (if it's not broken don't fix it), so I bumped this only one version, but I'm changing to to v4